cpp/data/system_updates/12.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+set -o errexit
+
+
+readonly HOSTNAME=$(hostname)
+if [[ $HOSTNAME == "ub16.uni-tuebingen.de" || $HOSTNAME == "ptah.ub.uni-tuebingen.de" ]]; then
```

Comment:
There are additional hostnames for which this should be changed, especially mb049.ub.uni-tuebingen.de and ub55.v264.uni-tuebingen.de. I think it would be better to check for TUEFIND_FLAVOUR=ixtheo instead.

Reply:
Done.